************* Module src.unit_test_generator
unit_test_generator.py:168:0: C0301: Line too long (115/100) (line-too-long)
unit_test_generator.py:169:0: C0301: Line too long (102/100) (line-too-long)
unit_test_generator.py:943:0: C0301: Line too long (120/100) (line-too-long)
unit_test_generator.py:947:0: C0301: Line too long (112/100) (line-too-long)
unit_test_generator.py:1891:0: C0301: Line too long (104/100) (line-too-long)
unit_test_generator.py:1895:0: C0301: Line too long (116/100) (line-too-long)
unit_test_generator.py:1978:0: C0301: Line too long (125/100) (line-too-long)
unit_test_generator.py:1:0: C0302: Too many lines in module (2330/1000) (too-many-lines)
unit_test_generator.py:1146:5: W0511: TODO use repr on the values? (fixme)
unit_test_generator.py:1173:5: W0511: TODO Add the function file and function name (fixme)
unit_test_generator.py:1183:9: W0511: TODO Return the cached result to save a lot of time! (fixme)
unit_test_generator.py:1801:9: W0511: TODO This is clearly wrong (fixme)
unit_test_generator.py:1888:9: W0511: TODO Why is this_state.args_after sometimes a tuple?? (fixme)
unit_test_generator.py:165:8: C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
unit_test_generator.py:226:4: R0914: Too many local variables (16/15) (too-many-locals)
unit_test_generator.py:890:0: R0902: Too many instance attributes (10/7) (too-many-instance-attributes)
unit_test_generator.py:894:4: R0913: Too many arguments (6/5) (too-many-arguments)
unit_test_generator.py:1969:49: W0123: Use of eval (eval-used)
unit_test_generator.py:2096:34: W0123: Use of eval (eval-used)
unit_test_generator.py:2312:4: W0105: String statement has no effect (pointless-string-statement)

------------------------------------------------------------------
Your code has been rated at 9.84/10 (previous run: 9.87/10, -0.02)

